File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1828_basico.js
import { question } from "readline-sync";

function main() {
    const T = get_Number();
    if(T <= 100) {
        for (let i = 1; i <= T; i++) {
            let escolha = get_Text().split(` `);
    
            if (escolha[0] === `tesoura`) {
                if (escolha[1] === `papel`) {
                    print(`Caso #${i}: Bazinga!`);
                } else if (escolha[1] === `lagarto`) {
                    print(`Caso #${i}: Bazinga!`);
                } else if (escolha[1] === `pedra`) {
                    print(`Caso #${i}: Raj trapaceou!`);
                } else if (escolha[1] === `Spock`) {
                    print(`Caso #${i}: Raj trapaceou!`);
                } else {
                    print(`Caso #${i}: De novo!`)
                }
            } else if (escolha[0] === `papel`) {
                if (escolha[1] === `pedra`) {
                    print(`Caso #${i}: Bazinga!`);
                } else if (escolha[1] === `Spock`) {
                    print(`Caso #${i}: Bazinga!`);
                } else if (escolha[1] === `tesoura`) {
                    print(`Caso #${i}: Raj trapaceou!`);
                } else if (escolha[1] === `lagarto`) {
                    print(`Caso #${i}: Raj trapaceou!`);
                } else {
                    print(`Caso #${i}: De novo!`)
                }
            } else if (escolha[0] === `pedra`) {
                if (escolha[1] === `lagarto`) {
                    print(`Caso #${i}: Bazinga!`);
                } else if (escolha[1] === `tesoura`) {
                    print(`Caso #${i}: Bazinga!`);
                } else if (escolha[1] === `papel`) {
                    print(`Caso #${i}: Raj trapaceou!`);
                } else if (escolha[1] === `Spock`) {
                    print(`Caso #${i}: Raj trapaceou!`);
                } else {
                    print(`Caso #${i}: De novo!`)
                }
            } else if (escolha[0] === `lagarto`) {
                if (escolha[1] === `Spock`) {
                    print(`Caso #${i}: Bazinga!`);
                } else if (escolha[1] === `papel`) {
                    print(`Caso #${i}: Bazinga!`);
                } else if (escolha[1] === `tesoura`) {
                    print(`Caso #${i}: Raj trapaceou!`);
                } else if (escolha[1] === `pedra`) {
                    print(`Caso #${i}: Raj trapaceou!`);
                } else {
                    print(`Caso #${i}: De novo!`)
                }
            } else if (escolha[0] === `Spock`) {
                if (escolha[1] === `tesoura`) {
                    print(`Caso #${i}: Bazinga!`);
                } else if (escolha[1] === `pedra`) {
                    print(`Caso #${i}: Bazinga!`);
                } else if (escolha[1] === `papel`) {
                    print(`Caso #${i}: Raj trapaceou!`);
                } else if (escolha[1] === `lagarto`) {
                    print(`Caso #${i}: Raj trapaceou!`);
                } else {
                    print(`Caso #${i}: De novo!`)
                }
            }
        }
    }
}

function get_Number() {
    return Number(question());
}

function get_Text() {
    return question();
}

function print(texto) {
    return console.log(texto);
}

main();

File: Exercicios_ED_I/Bee/1828_basico.cpp
#include <stdio.h>
#include <iostream>
#include <string.h>
using namespace std;

main() {
    int i, k;
    cin >> i;

    for (k = 1; k <= i; k++) {
        string jogada_1, jogada_2;
        cin >> jogada_1 >> jogada_2;

        if (jogada_1 == "tesoura") {
            if (jogada_2 == "papel") {
               printf("Caso #%d: Bazinga!\n", k);
            } else if (jogada_2 == "lagarto") {
               printf("Caso #%d: Bazinga!\n", k);
            } else if (jogada_2 == "pedra") {
               printf("Caso #%d: Raj trapaceou!\n", k);
            } else if (jogada_2 == "Spock") {
               printf("Caso #%d: Raj trapaceou!\n", k);
            } else {
               printf("Caso #%d: De novo!\n", k);
            }
        } else if (jogada_1 == "papel") {
            if (jogada_2 == "pedra") {
               printf("Caso #%d: Bazinga!\n",k);
            } else if (jogada_2 == "Spock") {
               printf("Caso #%d: Bazinga!\n", k);
            } else if (jogada_2 == "tesoura") {
               printf("Caso #%d: Raj trapaceou!\n", k);
            } else if (jogada_2 == "lagarto") {
               printf("Caso #%d: Raj trapaceou!\n", k);
            } else {
               printf("Caso #%d: De novo!\n", k);
            }
        } else if (jogada_1 == "pedra") {
            if (jogada_2 == "lagarto") {
               printf("Caso #%d: Bazinga!\n", k);
            } else if (jogada_2 == "tesoura") {
               printf("Caso #%d: Bazinga!\n", k);
            } else if (jogada_2 == "papel") {
               printf("Caso #%d: Raj trapaceou!\n", k);
            } else if (jogada_2 == "Spock") {
               printf("Caso #%d: Raj trapaceou!\n", k);
            } else {
               printf("Caso #%d: De novo!\n", k);
            }
        } else if (jogada_1 == "lagartoto") {
            if (jogada_2 == "Spock") {
               printf("Caso #%d: Bazinga!\n", k);
            } else if (jogada_2 == "papel") {
               printf("Caso #%d: Bazinga!\n", k);
            } else if (jogada_2 == "tesoura") {
               printf("Caso #%d: Raj trapaceou!\n", k);
            } else if (jogada_2 == "pedra") {
               printf("Caso #%d: Raj trapaceou!\n", k);
            } else {
               printf("Caso #%d: De novo!\n", k);
            }
        } else if (jogada_1 == "Spock") {
            if (jogada_2 == "tesoura") {
               printf("Caso #%d: Bazinga!\n", k);
            } else if (jogada_2 == "pedra") {
               printf("Caso #%d: Bazinga!\n", k);
            } else if (jogada_2 == "papel") {
               printf("Caso #%d: Raj trapaceou!\n", k);
            } else if (jogada_2 == "lagarto") {
               printf("Caso #%d: Raj trapaceou!\n", k);
            } else {
               printf("Caso #%d: De novo!\n", k);
            }
        }
    }
}